### PR TITLE
code/style:Fix spelling issue

### PR DESCRIPTION
**Pull request by ifraixedes**
~NOTE that the link didn't work browsing this docs through this repository nor the [Wiki](https://github.com/storj/storj/wiki/).~

~With this change, the link should work in the wiki but not thorough this repository, both at the same time aren't possible.~

Now, this PR only fixes a misspelling issue for being possible to merge without waiting to figure out which is the best option for links.

### CODE REVIEW

**Comment by stefanbenten** [comment]
I am not sure which way is preferred here.
Lets wait for @iglesiasbrandon or @jtolds

File: code/Database.md [hunk 1]
```diff
@@ -111,4 +111,4 @@ To modify the database the following steps must be completed:
 3. Add a new `testdata/*.v<N>.sql`, where `N` is the specific migration step.
 4. If there are new tables, then add a `-- NEW DATA --` and insert rows to the new tables.
 
-See [How to add a new table to Satellite.DB](Satellite-how-to-new-table.md) for more detailed instructions.
\ No newline at end of file
+See [How to add a new table to Satellite.DB](Satellite-how-to-new-table) for more detailed instructions.
```
Comment (JessicaGreben) [inline code/Database.md]: thanks for fixing

Comment (ifraixedes) [reply code/Database.md]: It depends, what we choose, I'm waiting for a response.

**Comment by egonelbre** [comment]
I would like both to work, but to do that we would need to drop the two-way-sync, use this repo links as ground truth and convert them to wiki format during sync.

**Pull request merged**